kore/test/Test/Kore/Builtin/List.hs

```diff
@@ -417,6 +419,91 @@ test_size =
         (===) Pattern.top    =<< evaluateT predicate
     ]
 
+test_make :: [TestTree]
```

Comment:
I don't think the new property tests are useful compared to having a list of simple test cases.